### PR TITLE
New DQM plots for CSV PF b tag trigger paths

### PR DESCRIPTION
**Pull request by HeinerTholen**
Adding new DQM plots for CSV PF b tag trigger paths. 

Here's the last presentation on the topic:
https://indico.cern.ch/event/700252/#3-trigger-dqm-update

It's planned to add the complementary implementation for Calo b tagging as well.

### CODE REVIEW

**Comment by jfernan2** [comment]
please test

File: DQMOffline/Trigger/src/BTVHLTOfflineSource.cc [hunk 1]
```diff
@@ -118,45 +121,50 @@ BTVHLTOfflineSource::analyze(const edm::Event& iEvent, const edm::EventSetup& iS
...
-    unsigned index = triggerNames_.triggerIndex(v.getPath()); 
-    if (index < triggerNames_.size() ){     
+    unsigned index = triggerNames_.triggerIndex(v.getPath());
+    if (index < triggerNames_.size() ){
      float DR  = 9999.;
+//                                                                                _|_|_|    _|_|_|_|
+//                                                                                _|    _|  _|
+//                                                                                _|_|_|    _|_|_|
+//                                                                                _|        _|
+//                                                                                _|        _|
```
Comment (dmitrijus) [inline DQMOffline/Trigger/src/BTVHLTOfflineSource.cc]: ?

**Comment by dmitrijus** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**